python/pyspark/sql/tests.py

```diff
@@ -234,6 +234,10 @@ def test_empty_row(self):
         row = Row()
         self.assertEqual(len(row), 0)
 
+    def test_row_without_column_name(self):
+        row = Row("Alice", 11)
```

Comment:
Can we add a doctest for this usage (Row as objects not as a namedtuple class), and documentation in `Row` at `types.py`?